### ISSUE TITLE
Can we see javascript "console.log" output

### ISSUE THREAD

**Issue by jrobinso** [googlecolab/colabtools#2644]
Is there anyway to see the "console.log()" output from Javascript in a Colab notebook?

I'm trying to port a module that works in Jupyter notebooks to Colab, it makes use of Comms, which needs modified to work with Colab (https://github.com/igvteam/igv-notebook).   Its failing and I can't see a way to set javascript breakpoints in Colab.   Simple console.log() statements would help but they go into the ether.   Is there anyway to see this output?  If not consider this a feature request.

I've searched for hours for an answer to this before posting, BTW.

**Comment by jrobinso** [comment]
Oh never mind now it works after fixing a bug.   Apparently something was blocking developer tools (neither console.log nore inspect element were working).   Either fixing my javascript bug enabled there, or it just coincidentally started working.

**Comment by blois** [comment]
Glad you figured it out- the devtools is indeed the typical way that I debug these.

**Comment by jrobinso** [comment]
@blois Thanks for the response.   I didn't really figure it out,  I guess I just managed to trash my VM somehow to the point devtools weren't workin.   I just thought, wrongly,  it was normal.    What I'm doing is unusual so there's not a lot of guidance or patterns.   I'm embedding a javascript visualization into a notebook cell that reads files in slices as needed, by URL (XMLHttpRequest) or from a browser "File" blob.     The URL loads just worked, as expected, but  I had to do something about local File objects.   I ended up emulating a File blob on the javascript side, delegating the actual reading to Python via google.colab.kernel.invokeFunction.   Its working now.
